src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -837,7 +837,7 @@ protected function send_debug_email() {
 
 If you think these failures might be due to a bug in ClassicPress, could you report it?
  * If you need general support: https://docs.classicpress.net/faq-support/
- * Or, if you're comfortable writing a bug report: https://docs.classicpress.net/testing-classicpress/#reporting-bugs
+ * Or, if you're comfortable writing a bug report: https://link.classicpress.net/testing-classicpress/#reporting-bugs
```

Comment:
As noted on the other PR, the `#hash-fragments` need to go away too. This one could probably be https://link.classicpress.net/reporting-bugs ...